src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -329,6 +330,18 @@ class ReimbursementAccountPage extends React.Component {
         const currentStep = achData.currentStep || CONST.BANK_ACCOUNT.STEP.BANK_ACCOUNT;
         const policyName = lodashGet(this.props.policy, 'name');
 
+        if (_.isEmpty(this.props.policy)) {
+            return (
+                <ScreenWrapper>
+                    <FullPageNotFoundView
+                        shouldShow
+                        onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS_WORKSPACES)}
```

Comment:
Coming from https://github.com/Expensify/App/issues/23167:
This caused infinite loop when this page is opened from deeplink.
`Navigation.goBack(ROUTES.SETTINGS_WORKSPACES)` is the correct approach.